### ISSUE TITLE
Change color of buttons

### ISSUE THREAD

**Issue by ventris** [prometheus/alertmanager#956]
I would like to explore the possibility to change the light blue button color in Alertmanager.
When its against a white background it's really straining for the eyes.

I have toyed around in the browsers developer tools and I find that the danger color-element from bootstrap is easier on the eyes than the info color-element.

**Comment by w0rm** [comment]
@mxinden let's get feedback from more people, because this doesn't look like smth immediately actionable. 

I'm definitely against of using `danger` for buttons, because the alert's buttons are less important than timestamps or tags. You first find the alert and only then click the button. Perhaps we can even use the secondary style if `info` is not an option.

**Comment by bluecamel** [comment]
I'm really not a fan of the new styles, in general.  It's much harder to glance at the alerts dashboard and see if things are firing (the old view, with red and green, was perfect for this).  Is there a simple way to change themes?

**Comment by stuartnelson3** [comment]
The new front end is a rewrite, the old dashboard does not exist anymore

**Comment by lae** [comment]
I have to agree with @bluecamel, I finally got around to refreshing our monitoring deployment (we were still on Prom 0.18.x) and the new grey on white color scheme is a bit strenuous on the eyes.
